pkg/util/util.go

```diff
@@ -71,5 +75,10 @@ func (dt *ThroughputCalc) Start() {
 // MaskPassword is used to mask sensitive password data before outputing to persistent stream like logs.
 func MaskPassword(s string) string {
 	s = maskPasswordRegex1.ReplaceAllString(s, maskPasswordReplaceString1)
-	return maskPasswordRegex2.ReplaceAllString(s, maskPasswordReplaceString2)
+	s = maskPasswordRegex2.ReplaceAllString(s, maskPasswordReplaceString2)
+	return maskPasswordRegex3.ReplaceAllString(s, maskPasswordReplaceString3)
 }
+
+func MaskPasswordinServiceURL(s string) string {
+	return maskPasswordRegex4.ReplaceAllString(s, maskPasswordReplaceString4)
+}
```

Comment:
Let's keep all the variants in one function `MaskPassword`. These are so seldom used performance doesn't matter and its safer if there is just one function.